### PR TITLE
Daniil/linter-update

### PR DESCRIPTION
**Pull request by AnkushinDaniil**
Makefile updated
Warnings fixed
Comments updated

### CODE REVIEW

File: .golangci.yaml [hunk 1]
```diff
@@ -156,11 +156,11 @@ issues:
       text: "SA1019: (.+).(GoVersion|LangVersion) is deprecated: use the global `run.go` instead."
     - path: pkg/golinters/unused.go
       text: "rangeValCopy: each iteration copies 160 bytes \\(consider pointers or indexing\\)"
-
-run:
-  timeout: 5m
```
Comment (kirugan) [inline .golangci.yaml:161]: do you know how `timeout` was used?

File: Makefile [hunk 2]
```diff
@@ -68,11 +68,11 @@ install-mockgen:
 	go install go.uber.org/mock/mockgen@latest
 
 install-golangci-lint:
-	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.56.2
+	go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.59.1
 
 lint:
 	@which golangci-lint || make install-golangci-lint
-	golangci-lint run
+	golangci-lint run
```
Comment (kirugan) [inline Makefile:75]: trailing whitespace?

File: .golangci.yaml (hunk 1 above)
Comment (AnkushinDaniil) [reply .golangci.yaml:161]: [Timeout](https://golangci-lint.run/usage/configuration/) for total work (default 1m0s)

File: Makefile (hunk 2 above)
Comment (AnkushinDaniil) [reply Makefile]: [fixed](https://github.com/NethermindEth/juno/commit/e0b12034bb1fa9809fb7bc2898d02f8ec77c2241)

**Review by kirugan — APPROVED** [review]

**Comment by kirugan** [comment]
> Is it possible to have a check to avoid having new warnings in the future?

@derrix060 what do you mean?

**Comment by kirugan** [comment]
@AnkushinDaniil please update `.github/workflows/juno-lint.yml` this file as well (version) that's why linter pipeline fails

**Comment by AnkushinDaniil** [comment]
[done](https://github.com/NethermindEth/juno/commit/08cd12a1426a5f2da068b77cb2ec347c8225b3c5)

**Comment by AnkushinDaniil** [comment]
> Is it possible to have a check to avoid having new warnings in the future?

The option is modify CI script to fail if there are warnings.

**Comment by kirugan** [comment]
btw those warnings was new ones because of the update

**Review by derrix060 — APPROVED** [review]

**Pull request merged**